numpy/linalg/linalg.py

```diff
@@ -1609,6 +1609,9 @@ def slogdet(a):
     ----------
     a : (..., M, M) array_like
         Input array, has to be a square 2-D array.
+        .. versionadded:: 1.8.0
```

Comment:
+1, Also some of the other linalg stuff has multiple arguments, so putting it for every single argument is probably too much anway.